lib/OpenQA/CacheService/Task/Asset.pm

```diff
@@ -2,17 +2,15 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 
 package OpenQA::CacheService::Task::Asset;
-use Mojo::Base 'Mojolicious::Plugin';
+use Mojo::Base 'Mojolicious::Plugin', -signatures;
 
 sub register {
     my ($self, $app) = @_;
```

Comment:
You skipped this one.

Reply:
See the comment I made above, I handled this similar to a problem I had in https://github.com/os-autoinst/openQA/pull/4476 in lib/OpenQA/CacheService/Plugin/Helpers.pm

The register method gets passed three values so I either skip it or add a @ at the end of the signature.

Reply:
The third argument should be `$conf`, no? According to [the docs](https://docs.mojolicious.org/Mojolicious/Plugin#register)?

Reply:
As I suggested in another PR, for subroutines like this (implementing a third-party API, or a third-party callback), it might be wise to use
`($self, $app, @)`
in this case, because we cannot guarantee that the third-party module will never add arguments.
Opinions?

Reply:
It's very unlikely Mojolicious will ever add another argument there. But if you want to be safe `($self, $app, @)` is not a bad choice.

Reply:
I added a $conf parameter for the two subroutines in question and also went back to do the same for the Helpers.pm from before.
I don't like the approach of adding an `·@` because it seems to go against the [reason for changing to signatures](https://progress.opensuse.org/issues/99579)

Reply:
It's still adding value for the first parameters that are checked for their type, at least somewhat